doc/data/messages/d/duplicate-bases/related.rst

```diff
@@ -0,0 +1 @@
+`Issue Tracker <https://github.com/PyCQA/pylint/issues?q=is%3Aissue+%22inconsistent-mro%22+OR+%22E0240%22>`_
```

Comment:
See my comments on the other PR you openend :)

Reply:
Thank you ;)